spyder/plugins/ipythonconsole.py

```diff
@@ -877,6 +871,13 @@ def create_new_client(self, give_focus=True):
                               menu_actions=self.menu_actions)
         self.add_tab(client, name=client.get_name())
 
+        if cf is None:
+            error_msg = ("The directory {} is not writable and it is "
+                         "required to create IPython consoles. Please make "
+                         "it writable.").format(jupyter_runtime_dir())
+            client.show_kernel_error(_(error_msg))

```

Comment:
After the previous change, this only needs to be

    client.show_kernel_error(error_msg)